modules/apps/asset/asset-tags-service/src/main/java/com/liferay/asset/tags/internal/search/spi/model/index/contributor/AssetTagDocumentContributor.java

```diff
@@ -88,6 +89,10 @@ private void _contributeAssetTagNamesRaw(
 		Document document, List<AssetTag> assetTags) {
 
 		document.addText(Field.ASSET_TAG_NAMES, _getNames(assetTags));
+		document.addText(
+			"lowerCaseAssetTagNames",
```

Comment:
@4lejandrito, @ccorreagg, I was taking a look at the other mappings, and I'm unsure if Brian will like "lowerCase.....etc" 😓

Reply:
No longer applicable!